### ISSUE TITLE
_property-of

### ISSUE THREAD

**Issue by ellioseven** [davidkpiano/sassdash#26]
First off, I love your work. This library is awesome.

I am going through your documentation and I am trying to work with `_property-of()`. I am getting different results however.

``` scss
$map: ( a: 3, b: 1, c: 2 );
$foo: _map((a, c), _property-of($map));
@debug $foo;
// Documentation => (3, 2)
// Me => null null
```

I am not sure if I am doing something wrong here?

This could be an issue with my current stack however, as I am using `gulp-sass` to compile my stylesheets.

**Comment by ellioseven** [comment]
A further note, I can't seem to find any unit tests for this function?

**Comment by davidkpiano** [comment]
Sorry about that! I'll add unit tests and publish a patch.

**Comment by davidkpiano** [comment]
Published as `0.8.2`
